app/uk/gov/hmrc/ctcguaranteebalanceeisstub/models/responses/BalanceResponse.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ */
+
+package uk.gov.hmrc.ctcguaranteebalanceeisstub.models.responses
+
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.json.Format
+import play.api.libs.json.Json
+import play.api.libs.json.__
+import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.AccessCode
+import uk.gov.hmrc.ctcguaranteebalanceeisstub.models.GuaranteeReferenceNumber
+
+import scala.util.Random
+
+case class BalanceResponse(guaranteeReferenceNumber: GuaranteeReferenceNumber, remainingBalance: Balance)
+
+case class Balance(value: Int) extends AnyVal
```

Comment:
Balance needs to be a double really, truncated to two decimal places.

Reply:
That's what I thought initially but it said a number in the ticket so confused me a bit. Shall update